rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -548,14 +548,14 @@ void RSXThread::update_reg(u32 reg, u32 value)
 
 	case NV4097_SET_CULL_FACE:
 	{
-		m_cull_face = value;
+		m_cull_face = (value == 0x0404) || (value == 0x0405) || (value == 0x0408) ? value : 0x0405/*default GL_BACK*/;
```

Comment:
Is that the way the API behaves? Because if that's not how it works in the real PS3 this might mask other bugs. Maybe just logging if it's not a valid value is more apropriate